### PR TITLE
feat(herbs2mermaid) - implement herbs2mermaid to render usecases diagram

### PR DESCRIPTION
**Pull request by maikmb**
Fixes #

#37 

## Proposed Changes

1. Add herbs2mermaid to render usecases diagram
2. Usecase dark/light mode in graphs

### Author/Contributor
- [X] If documentation is needed for this change, has that been included in this pull request
- [X] Remember to check if code coverage decrease, if so, please implement the necessary tests

### Reviewing Maintainer
- [X] Label as `breaking` if this is a large fundamental change
- [ ] Label as either `automation`, `bug`, `documentation`, `enhancement`, `infrastructure`, or `performance`

### CODE REVIEW

File: src/shelf.js [hunk 1]
```diff
@@ -76,10 +78,15 @@ function herbsshelf({ herbarium, project, description = '', readmePath = './READ
...
 		entity: item.entity,
 		id: item.id,
-		metadata: item.metadata
+		tags: { group: item.group || 'Others' }
 	}))
 
-	const specs = Array.from(herbarium.specs.all).map(([_, item]) => ({ spec: item.spec, id: item.id, usecase: item.usecase }))
+	const specs = Array.from(herbarium.specs.all).map(([_, item]) => ({
+		spec:
+			item.spec,
```
Comment (italojs) [inline src/shelf.js:86]: fix indentation here

File: test/shelf.test.js [hunk 2]
```diff
@@ -97,7 +97,7 @@ describe('Generate usecase self-documentation', () => {
 
 		it('should generate some doc', () => {
 			//given
-			usecases.push({ usecase: givenTheSimplestUseCaseWithRequestResponse(), tags: { group: 'SimplestestUseCaseWithRequestResponse' } })
+			usecases.push({ id: 'AUsecaseWithResponse', usecase: givenTheSimplestUseCaseWithRequestResponse(), tags: { group: 'SimplestestUseCaseWithRequestResponse' } })
```
Comment (italojs) [inline test/shelf.test.js:100]: this string is being repeated along this file, may be interesting create a const for it

**Review by italojs — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -70,6 +70,7 @@ const shelf = renderShelfHTML('Project Name', usecases(), entities, './custom-re
 ```
 
 
+
```
[the remark this answers is not in the document]
Comment (maikvortx) [reply README.md]: done!

File: src/shelf.js (hunk 1 above)
Comment (maikvortx) [reply src/shelf.js]: done!

File: test/shelf.test.js (hunk 2 above)
Comment (maikvortx) [reply test/shelf.test.js:100]: I think it's ok! The `usecases` variable is declared in different contexts - describe.

**Pull request merged**